Automattic/jetpack
# Open Graph: update fallback images to return better images.

#### Testing instructions:

Test on a site using Jetpack for its Open Graph Meta tags:
- Make sure the OG tags, especially the `og:image`, `og:image:width`, and `og:image:height` are always correct on the home page:
    - When only using a Site Icon under Appearance > Customize > Site Identity
    - When using a theme logo when your theme supports it.
- The images used should always be the largest we have available, not just the minimum size (200x200).

#### Proposed changelog entry for your changes:
* Sharing: update Open Graph Image tags appearing on the home page to offer better alternatives based on your site settings in Appearance > Customize.
